declarations/Instagram.json

```diff
@@ -8,6 +8,71 @@
     "Terms of Service": {
       "fetch": "https://help.instagram.com/581066165581870",
       "select": [".bk"]
+    },
+    "Community Guidelines": {
+      "fetch": "https://help.instagram.com/477434105621119",
+      "select": [".bj", ".bl"]
+    },
+    "Community Guidelines - Self-harm": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/suicide-self-injury/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Hate Speech": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/hate-speech/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Child Sexual Exploitation": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/child-sexual-exploitation-abuse-nudity/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Violence Incitement": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/violence-incitement/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Violent Organizations": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/dangerous-individuals-organizations/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Spam": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/spam/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Regulated Goods": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/regulated-goods/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Harassment": {
+      "fetch": "https://transparency.fb.com/fr-fr/policies/community-standards/bullying-harassment/",
+      "select": ["._9nrm", "._9q49", "._9p7c"],
+      "remove": ["._9p72", "svg", "._9ooi", "._9q3_"]
+    },
+    "Community Guidelines - Intellectual Property": {
```

Comment:
Should be `Copyright Policy`.

Reply:
Mmh no sorry, there is no such type… I mixed it up with `Copyright Claims Policy` 🙃

Reply:
Fixed in https://github.com/OpenTermsArchive/declarations-france-elections/pull/12